app/styles/_add-config-to-application.less

```diff
@@ -52,6 +52,14 @@
     margin-bottom: 10px;
   }
 
+  .key {
+    white-space: nowrap;
+    .pficon-warning-triangle-o {
```

Comment:
Can you try reusing this style instead of adding a new one?

https://github.com/spadgett/origin-web-console/blob/0a97382207e3c1d1d85657abde47f01a43145c84/app/styles/_tooltip.less#L6-L9

I think we need one common tooltip style so we're consistent everywhere